styles/globals.css

```
@@ -11,81 +11,20 @@
   }
 }
 
-/* Inter font family with multiple weights for better Safari compatibility */
+/* Inter variable font — single file covers all weights (100–900) */
 @font-face {
   font-family: "Inter";
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)